.github/workflows/escalation-slack-notification.yml

```
@@ -10,6 +10,12 @@ jobs:
     runs-on: ubuntu-22.04
     timeout-minutes: 10
     steps:
+      - name: Setting issue title and quote escaping
```

Nit:
Can we please use the imperative here in order to keep step names uniform?
```suggestion
      - name: Format issue title (escape quotes)
```

^ or anything similar to this.

Feel free to submit the change with the `[ci skip]` and we can force-merge it to avoid wasting another CI cycle.